hastory-cli/src/Hastory/Cli/OptParse.hs

```diff
@@ -5,29 +5,39 @@ module Hastory.Cli.OptParse
   , Instructions(..)
   , Dispatch(..)
   , Settings(..)
+  , combineToInstructions
   ) where
 
 import Data.Maybe (fromMaybe)
 import Data.Semigroup ((<>))
 import qualified Data.Text as T
 import Options.Applicative
 import Path.IO (getHomeDir, resolveDir, resolveDir')
-import Servant.Client.Core.Reexport (parseBaseUrl)
-import System.Environment (getArgs)
+import Servant.Client.Core (parseBaseUrl)
+import qualified System.Environment as System
 
 import Data.Hastory.API (Token(..))
 import Hastory.Cli.OptParse.Types
 
+getEnvironment :: IO Environment
```

Comment:
Nit: Usually we put `getInstructions` and `combineToInstructions` at the top of the file.